core/src/main/resources/org/elasticsearch/bootstrap/security.policy

```diff
@@ -92,7 +92,9 @@ grant {
   permission java.lang.RuntimePermission "accessClassInPackage.sun.reflect";
   // needed by aws core sdk (TODO: look into this)
   permission java.lang.RuntimePermission "accessClassInPackage.sun.security.ssl";
-  
+  // needed by GCE tests sdk
+  permission java.lang.RuntimePermission "accessClassInPackage.sun.util.calendar";
```

Comment:
I'd love to! Any idea on how I can find which class is responsible for those errors ?